back.py

```diff
@@ -76,7 +77,19 @@ def run():
     logger = MeticulousLogger.getLogger(__name__)
 
     try:
-        update_db_migrations()
+        try:
+            ShotManager.init()
+            logger.info("ShotManager initialized successfully")
```

Comment:
I think I would prefer this in its own function, maybe at the top of shotDatabase.init? Looks good otherwise. Thank you